frontstage/views/sign_in/sign_in.py

```diff
@@ -91,8 +91,8 @@ def login():  # noqa: C901
         response.set_cookie('authorization',
                             value=session.session_key,
                             expires=session.get_expires_in(),
-                            secure=secure,
-                            httponly=secure)
+                            secure=True,
```

Comment:
I think this is configurable for a reason??

The secure variable is set by the WTF_CSRF_ENABLED environment variable and it won't make a difference in preprod/prod as it's enabled, so I'm not sure why you're doing this?

Reply:
You're right: reverting.